README.md

```diff
@@ -261,6 +261,8 @@ about getting involved in the community:
 
 ## Contributing
 
+[![GitHub repo Good Issues for newbies](https://img.shields.io/github/issues/kubeflow/katib/good%20first%20issue?style=flat&logo=github&logoColor=green&label=Good%20First%20issues)](https://github.com/kubeflow/katib/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) [![GitHub Help Wanted issues](https://img.shields.io/github/issues/kubeflow/katib/help%20wanted?style=flat&logo=github&logoColor=b545d1&label=%22Help%20Wanted%22%20issues)](https://github.com/kubeflow/katib/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) [![GitHub Help Wanted PRs](https://img.shields.io/github/issues-pr/kubeflow/katib/help%20wanted?style=flat&logo=github&logoColor=b545d1&label=%22Help%20Wanted%22%20PRs)](https://github.com/kubeflow/katib/pulls?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) [![GitHub repo Issues](https://img.shields.io/github/issues/kubeflow/katib?style=flat&logo=github&logoColor=red&label=Issues)](https://github.com/kubeflow/katib/issues?q=is%3Aopen)
```

Comment:
Sorry for the late review @devactivity-team!
I think, it is a great enhancement for the contributing section.
What do you think @kubeflow/wg-training-leads ?